usmqe/api/tendrlapi/common.py

```diff
@@ -215,6 +215,78 @@ def get_nodes(self):
         self.check_response(response)
         return response.json()
 
+    def create_cluster(
+            self,
+            name,
+            cluster_id,
+            nodes,
+            provisioner,
+            public_network,
+            cluster_network,
+            node_identifier,
+            conf_overrides,
+            sds_type,
+            sds_version,
+            asserts_in=None):
+        """Create cluster.
+
+        Name:        "create_cluster",
+        Method:      "POST",
+        Pattern:     "CreateCluster",
+
+        Args:
+            name (str): name of cluster
+            cluster_id (str): id of cluster
+            nodes (list): list of dictionaries containing node identification
+                        and node role
+            provisioner (str): node identification of node that contain
+                        provisioning tag
+            public_network (str): ip address and mask in prefix format
+                        of network with nodes
+            cluster_network (str): ip address and mask in prefix format
+                        of network with nodes
```

Comment:
The description of public and cluster network is the same here, this doesn't sound right :)